examples/generic-azure.yaml

```diff
@@ -673,3 +673,26 @@ groups:
           ## References
           - section 9.1.2 https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/
         scored: true
+      - id: 1.5.3
+        description: "<sid>adm user and its <sid> group exist and enabled"
+        audit: |
+          sid=$(sudo crm configure show | grep -m1 SID= | sed -e "s/.*SID=\(...\).*/\1/" | tr '[:upper:]' '[:lower:]')
+          if cat /etc/passwd | cut -d: -f1 | grep ${sid}adm; then echo userexists; fi
```

Comment:
@aleksei-burlakov is it intended, that the 'sid'adm user name is displayed in addition to the 'userexists'?
Because of the 'grep' command you will get something like
prdadm
userexists
as the result.

Reply:
It's ok, that the user name is displayed, it will be ignored.

Reply:
At least with "grep -q" you could easy remove the <sid>adm from the output which would be more clean from my point of view.